test/config/integration/server_unix_listener.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
Can you make this a v2 YAML? We're trying to avoid adding any new v1 configs to integration tests. Thanks.

Reply:
I started out with v2 yaml, then went to v1 json because the test helpers aren't compatible with the v2 configs at all.